build.sbt

```diff
@@ -52,8 +52,7 @@ lazy val catsDependencies = List(
   )
 
 libraryDependencies ++= Seq(
-  "org.broadinstitute" %% "wdl4s-wdl" % wdl4sV,
-  "org.broadinstitute" %% "wdl4s-cwl" % wdl4sV,
+  "org.broadinstitute" %% "wdl4s" % wdl4sV,
```

Comment:
Good point, I'll fix this up after the release has happened (and there's a stable version to point to)